core/tests/test_utils.py

```diff
@@ -59,6 +62,34 @@ def empty_environ():
     os.environ['DEFAULT_VERSION_HOSTNAME'] = '%s:%s' % (
         os.environ['HTTP_HOST'], os.environ['SERVER_PORT'])
 
+class URLFetchServiceMock(apiproxy_stub.APIProxyStub):
```

Comment:
Some tests don't work with this urlfetch mock. So this mock is enabled only on signup, and disabled immediately once the signup is completed.